chapters/background.tex

```diff
@@ -1,4 +1,14 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-
+Regarding soHappy, TesnorFlow \cite{tensorflow} appears to be a convenient fit when it comes to smile detection.
```

Comment:
"TesnorFlow" -> "TensorFlow"

"appears to be" sounds a bit too unconfident, i. e. like we're not entirely sure we know what we are doing.